lnst/Common/Parameters.py

```diff
@@ -179,6 +180,27 @@ def type_check(self, value):
         return value
 
 
+class NetworkParam(Param):
```

Comment:
Either provide some details in the commit and/or add a docstring for the new classes as this provides new API to the user.

If you docstring for these they will automatically appear in https://lnst.readthedocs.io/en/latest/parameters.html

Reply:
fixedup the commit history during rebase

Reply:
Unresolving. Neither commit has been updated with the description how this can be used, nor docstring was added.

Reply:
opps sorry resolved the wrong one.

Reply:
Added a docstring, resolving this thread for now, we can discuss it in other threads